slsDetectorServers/eigerDetectorServer/slsDetectorFunctionList.c

```diff
@@ -2548,8 +2548,18 @@ void *start_timer(void *arg) {
                 }
                 break;
             case 16:
+#ifdef TEST_MOD_GEOMETRY
```

Comment:
Should this be in the final code? if so maybe a comment what the ifdef does?

Reply:
added comment